### ISSUE TITLE
Cube stuck in the ground

### ISSUE THREAD

**Issue by unixpickle** [Unity-Technologies/obstacle-tower-env#80]
If you start at floor 0 of seed 56, and reach floor 22, you will see a box _stuck in the floor_. When you jump on the box, you shoot into the air above the ceiling. If you initialize the environment to floor 22, the issue does not occur.

[box_bug.mp4.zip](https://github.com/Unity-Technologies/obstacle-tower-env/files/3070971/box_bug.zip)

I also witnessed this on another seed some time ago, but don't have the seed number. Here's a picture:

![uh](https://user-images.githubusercontent.com/406021/56001758-94eab800-5c8f-11e9-9027-1e03f402cf66.png)

**EDIT:** also happens on floor 21 of seed 43.

**Comment by unixpickle** [comment]
One of my famous reproduction scripts, working on version 1.3 of the environment.

```python
from obstacle_tower_env import ObstacleTowerEnv

ACTIONS = [0]*5 + [18]*92 + [30]*5 + [18]*45 + [21]*2 + [18]*2 + [30]*4 + [18]*5 + [30] + [18]*16 + [21] + [18]*3 + [24]*5 + [18]*50 + [21] + [18] + [24]*4 + [18]*9 + [24] + [18]*40 + [21]*2 + [18]*87 + [24]*2 + [18] + [30]*8 + [18]*3 + [30]*10 + [18]*14 + [24] + [18]*43 + [30]*6 + [18]*4 + [24]*2 + [18]*4 + [30] + [18]*4 + [24] + [18]*32 + [24]*4 + [18]*3 + [24] + [18]*20 + [24]*7 + [18] + [30]*2 + [18]*67 + [30]*4 + [18]*8 + [30]*2 + [18]*3 + [24]*3 + [18]*2 + [30]*2 + [18]*7 + [21] + [33] + [18]*2 + [0]*3 + [12]*2 + [15] + [18]*2 + [30] + [18]*29 + [30]*3 + [18]*4 + [24]*2 + [18]*2 + [30] + [12]*5 + [30]*2 + [18]*4 + [24]*3 + [18]*3 + [24]*2 + [18]*3 + [30]*2 + [18]*4 + [24]*7 + [18]*3 + [30]*2 + [18]*3 + [24] + [18]*10 + [21]*2 + [18]*31 + [24]*4 + [18]*5 + [24] + [18]*16 + [24]*5 + [18]*23 + [30]*7 + [18]*2 + [24]*2 + [18]*45 + [24]*6 + [18]*4 + [30] + [18]*45 + [30]*8 + [18] + [24]*3 + [18]*11 + [30] + [18]*2 + [30]*3 + [18]*2 + [24]*13 + [18]*5 + [30] + [24]*3 + [6]*3 + [24] + [18]*2 + [30]*5 + [18]*39 + [24]*4 + [18]*7 + [24] + [18]*41 + [24]*3 + [30]*5 + [18]*2 + [24]*2 + [18]*3 + [30]*2 + [24]*2 + [18]*48 + [30]*10 + [18]*3 + [24]*5 + [18]*17 + [24]*6 + [18]*4 + [30]*2 + [18]*5 + [24] + [18]*5 + [24]*2 + [18] + [30]*2 + [18]*3 + [30]*2 + [18] + [6]*4 + [18] + [30]*2 + [12]*2 + [30] + [18]*11 + [24] + [18]*11 + [24]*6 + [18] + [30]*3 + [18]*8 + [30] + [18]*7 + [21]*2 + [18]*85 + [24]*3 + [18]*4 + [24]*2 + [18]*16 + [24]*2 + [18]*2 + [24] + [18]*2 + [24]*2 + [18]*18 + [30]*2 + [18] + [24]*4 + [6]*6 + [0]*3 + [6]*2 + [12]*4 + [18]*2 + [24] + [6]*3 + [24]*2 + [18]*5 + [24] + [18]*3 + [30]*2 + [18]*3 + [24]*2 + [18]*4 + [30] + [18]*4 + [24] + [6]*2 + [24]*3 + [18]*3 + [30]*3 + [18]*10 + [21] + [18]*3 + [30]*4 + [18]*49 + [30]*2 + [18]*3 + [24]*8 + [6]*2 + [24]*3 + [6] + [0] + [18]*5 + [24] + [18]*5 + [30]*2 + [18] + [24]*5 + [6]*2 + [24]*3 + [18] + [24]*3 + [6]*2 + [24]*3 + [12]*2 + [30]*10 + [12]*6 + [18]*3 + [24] + [18]*6 + [24]*2 + [18]*3 + [30] + [18]*11 + [30]*10 + [18]*10 + [24]*3 + [18]*3 + [30]*2 + [18]*4 + [24]*2 + [21]*2 + [18]*2 + [30] + [18]*2 + [0]*2 + [21] + [18] + [30] + [12]*4 + [18]*30 + [30]*6 + [18] + [24]*2 + [18]*2 + [30]*3 + [18]*3 + [24]*3 + [18] + [30]*2 + [18]*3 + [24]*2 + [18] + [30] + [18] + [0] + [18]*2 + [24] + [18]*2 + [30] + [18]*3 + [24] + [18]*2 + [30] + [18]*14 + [30]*4 + [18]*3 + [30]*2 + [18]*5 + [24] + [18]*6 + [30] + [18]*2 + [21]*2 + [18]*2 + [24] + [18]*11 + [30] + [18]*5 + [24] + [18]*7 + [30] + [18]*3 + [21] + [18] + [24]*2 + [18]*2 + [12]*2 + [15] + [18]*2 + [30] + [12]*3 + [18]*5 + [6]*4 + [18] + [30]*6 + [18] + [24]*3 + [18]*3 + [30] + [18]*5 + [30] + [18] + [6]*4 + [24] + [18]*3 + [24]*4 + [30]*6 + [24]*3 + [18]*3 + [24] + [18]*2 + [30] + [18]*6 + [24]*5 + [18]*5 + [30]*2 + [24]*4 + [18] + [30] + [18]*6 + [30] + [18]*3 + [30]*3 + [24]*8 + [6]*2 + [24]*5 + [6] + [24] + [18]*3 + [30]*2 + [18]*18 + [24]*8 + [18] + [30]*5 + [18]*4 + [24] + [18]*3 + [30] + [18]*3 + [24] + [18]*4 + [30] + [18]*4 + [24] + [18]*12 + [21]*2 + [18]*10 + [21] + [18]*11 + [21] + [18]*22 + [30] + [18]*3 + [24]*5 + [6]*2 + [0] + [18]*2 + [30]*2 + [12]*5 + [30] + [18]*5 + [30]*2 + [18]*6 + [24] + [18]*7 + [30]*5 + [18] + [24]*3 + [18] + [30]*3 + [24]*3 + [18]*7 + [21]*2 + [18]*2 + [30]*3 + [18]*3 + [30] + [18]*2 + [30]*8 + [0]*2 + [18]*2 + [24]*4 + [21]*2 + [18]*3 + [6]*6 + [3] + [18]*2 + [24]*2 + [18]*4 + [30] + [18]*4 + [30]*4 + [18]*12 + [24]*4 + [18]*3 + [24] + [18]*20 + [24]*6 + [18] + [24]*2 + [6]*4 + [24]*2 + [18]*2 + [30]*4 + [18]*3 + [24] + [18] + [30] + [18]*4 + [24] + [18] + [30] + [18]*6 + [24]*4 + [18]*5 + [24] + [18]*72 + [24]*4 + [18]*4 + [24]*2 + [6]*4 + [0]*2 + [6]*4 + [24] + [18]*2 + [24]*3 + [18] + [30]*2 + [18]*2 + [21]*2 + [18]*2 + [24] + [18]*5 + [6]*8 + [21] + [18]*5 + [24]*3 + [18]*7 + [30]*3 + [18]*17 + [21]*2 + [18]*3 + [30]*5 + [18]*5 + [24] + [18]*7 + [30]*2 + [18]*4 + [24]*2 + [18]*2 + [30] + [18]*3 + [30]*4 + [18] + [24]*3 + [18]*10 + [24]*3 + [6]*8 + [24] + [18] + [12]*3 + [30] + [18]*25 + [30]*7 + [18]*4 + [30]*2 + [18] + [24] + [18]*2 + [24]*2 + [18]*5 + [30]*2 + [21] + [18]*3 + [24] + [18]*27 + [21] + [18]*8 + [21] + [18]*33 + [21] + [18]*15 + [21] + [18]*24 + [21] + [18]*8 + [30]*4 + [18]*6 + [24]*6 + [18] + [24]*2 + [18]*2 + [24]*2 + [18] + [30]*6 + [12]*6 + [18]*5 + [30]*2 + [18] + [24]*2 + [18]*8 + [6]*3 + [24]*9 + [18]*3 + [24]*6 + [18]*4 + [12]*6 + [30] + [18] + [6]*3 + [24]*2 + [6]*4 + [24] + [18]*12 + [30]*2 + [18]*6 + [0] + [6]*7 + [18]*2 + [30] + [12]*3 + [30]*2 + [12]*5 + [18] + [24]*3 + [18]*2 + [30]*2 + [18]*8 + [30] + [18]*11 + [30]*2 + [12]*3 + [18]*2 + [21] + [18]*18 + [24] + [18]*12 + [21]*2 + [18]*11 + [21] + [18]*4 + [6]*8 + [24]*2 + [18]*29 + [21] + [18]*37 + [21]*2 + [18]*5 + [24]*3 + [6]*2 + [18] + [21] + [18]*24 + [21]*2 + [18]*39 + [24]*4 + [18]*3 + [24] + [18]*17 + [21] + [18]*4 + [0]*4 + [3] + [0] + [18]*55 + [30]*4 + [12]*4 + [30] + [18]*4 + [24] + [18]*2 + [24]*6 + [6]*4 + [18]*5 + [30]*2 + [18]*33 + [30] + [12]*8 + [18]*2 + [24]*2 + [18]*2 + [24]*2 + [18]*39 + [24]*5 + [18]*18 + [21]*2 + [18]*2 + [24]*5 + [18]*21 + [6]*3 + [0] + [12]*4 + [18]*11 + [21]*2 + [18] + [30]*3 + [18]*4 + [24] + [18] + [12]*7 + [21]*2 + [18]*6 + [24]*5 + [18]*3 + [24] + [18]*3 + [30] + [18]*5 + [30] + [18]*4 + [6]*6 + [18] + [21] + [18]*26 + [24] + [18] + [6] + [24]*5 + [18]*4 + [24]*5 + [6]*5 + [18]*16 + [24] + [18]*7 + [30] + [18]*11 + [24] + [18]*3 + [30]*2 + [18]*3 + [24]*2 + [18]*14 + [0] + [6]*3 + [24]*6 + [18]*14 + [30]*2 + [18]*17 + [30]*7 + [18]*3 + [24]*2 + [18]*10 + [24]*2 + [18]*5 + [24]*2 + [18]*3 + [24]*2 + [18]*4 + [30] + [18]*7 + [30]*3 + [18]*6 + [30] + [12]*9 + [30]*2 + [18]*25 + [0]*3 + [12]*4 + [18]*2 + [21]*2 + [24] + [18]*13 + [30]*2 + [18]*8 + [0]*2 + [6]*3 + [24] + [18]*3 + [33] + [30] + [18]*5 + [24]*2 + [18]*5 + [24] + [18]*11 + [30]*3 + [18]*5 + [24] + [18]*3 + [24]*5 + [6]*4 + [0] + [18]*21 + [24] + [18]*2 + [30]*2 + [18]*13 + [30]*2 + [12]*3 + [30] + [18]*2 + [24]*2 + [6]*6 + [24] + [18]*4 + [24] + [18]*9 + [30]*8 + [18] + [24]*3 + [18]*8 + [0] + [6] + [0] + [12]*5 + [18]*2 + [21] + [18]*4 + [24] + [18]*10 + [30] + [18]*44 + [21] + [18]*14 + [0] + [6]*5 + [18]*3 + [30]*2 + [12]*2 + [30]*2 + [12]*5 + [30] + [18]*3 + [30] + [18]*2 + [24] + [18]*4 + [24] + [18]*6 + [24] + [18]*10 + [6]*4 + [24]*2 + [18] + [30] + [12]*3 + [30] + [12]*5 + [18]*6 + [30]*2 + [18]*6 + [12]*6 + [30]*2 + [18] + [24]*4 + [18]*4 + [21] + [18]*25 + [12]*3 + [6]*11 + [24]*2 + [18]*7 + [24] + [18]*15 + [24]*3 + [6]*3 + [24] + [18]*4 + [30]*2 + [18]*4 + [30] + [18]*5 + [0] + [12]*3 + [6]*5 + [24] + [18]*3 + [30] + [33] + [21] + [18]*3 + [21] + [18]*4 + [21] + [18]*8 + [21] + [18]*5 + [24]*3 + [18]*2 + [24] + [18]*7 + [0]*3 + [18]*5 + [21]*2 + [18]*11 + [30] + [21] + [18]*3 + [6] + [24]*2 + [18] + [0] + [6]*8 + [24] + [18]*5 + [30] + [21]*2 + [18]*7 + [21] + [18]*2 + [30] + [18] + [0] + [18]*7 + [24] + [18]*6 + [0]*2 + [6]*3 + [0]*3 + [18]*5 + [21] + [18]*13 + [33] + [30] + [18]*3 + [24]*4 + [18]*7 + [0]*3 + [6]*4 + [0] + [18]*5 + [30]*3 + [18]*7 + [30]*2 + [18]*9 + [30] + [18]*7 + [24] + [18]*3 + [24] + [18]*5 + [0] + [18]*4 + [30]*4 + [18] + [24]*7 + [18]*4 + [30]*3 + [18]*51 + [12]*4 + [18]*4 + [24]*3 + [18]*7 + [24]*2 + [18]*4 + [24]*2 + [18]*4 + [30]*2 + [18]*3 + [30] + [18]*11 + [6] + [24]*2 + [18]*3 + [24]*2 + [18]*5 + [30] + [18]*3 + [24] + [18]*14 + [24]*2 + [18] + [30] + [12]*4 + [30] + [18] + [24] + [6]*4 + [24]*5 + [18]*3 + [30] + [18]*13 + [0] + [12]*5 + [18]*7 + [24] + [18] + [21]*2 + [18]*5 + [24]*2 + [18]*2 + [21] + [24] + [18]*5 + [24] + [18]*2 + [21]*2 + [18] + [30]*2 + [18] + [30]*2 + [18]*5 + [30] + [18]*7 + [0]*5 + [18] + [30]*2 + [18]*3 + [6]*5 + [18]*2 + [0] + [6]*2 + [18] + [21] + [18]*24 + [0] + [12]*2 + [6]*3 + [0]*3 + [12]*2 + [0]*4 + [18]*2 + [21] + [24]*2 + [18]*2 + [0]*11 + [18] + [12]*2 + [0]*3 + [6]*4 + [0]*13 + [6] + [0]*6 + [18] + [21] + [18]*4 + [24] + [18] + [0] + [12]*4 + [21]*2 + [18]*4 + [24]*3 + [18]*4 + [24] + [18]*15 + [24]*3 + [18] + [6]*12 + [12]*4 + [0] + [18]*6 + [30] + [18]*13 + [0] + [12]*3 + [0]*8 + [6]*3 + [0]*2 + [18]*2 + [0]*2 + [12]*4 + [0]*4 + [18] + [21]*2 + [30] + [18]*5 + [0] + [18] + [21] + [18]*9 + [6]*4 + [0] + [18] + [21]*2 + [18]*5 + [30]*3 + [18]*4 + [30]*2 + [18]*6 + [6]*2 + [18]*8 + [30] + [0] + [18]*4 + [21] + [18]*8 + [21] + [18]*23 + [21] + [18]*4 + [30]*2 + [18] + [24]*3 + [18]*16 + [30] + [12]*4 + [30] + [18] + [6]*4 + [24]*2 + [6]*2 + [24]*2 + [18]*9 + [30]*6 + [18]*4 + [30]*2 + [18]*2 + [24]*3 + [18]*8 + [0]*3 + [18]*18 + [21] + [18]*20 + [24]*3 + [18] + [30]*10 + [18]*4 + [24]*3 + [18]*3 + [30]*2 + [18] + [24]*2 + [18]*4 + [30] + [18]*4 + [30]*5 + [18]*3 + [24]*2 + [18]*2 + [30] + [18]*5 + [30] + [18]*7 + [24]*4 + [18]*2 + [12]*9 + [30] + [18]*4 + [24]*3 + [18]*3 + [30] + [18]*4 + [30] + [18]*9 + [0]*2 + [12] + [6]*2 + [0]*8 + [18]*3 + [30] + [18]*2 + [24]*2 + [18] + [21]*2 + [18]*4 + [30] + [18]*3 + [21] + [18]*5 + [6]*5 + [24]*3 + [18]*7 + [30]*3 + [18]*5 + [30] + [0] + [18]*30

env = ObstacleTowerEnv('./ObstacleTower/obstacletower', worker_id=0)
env.seed(56)
env.reset()
for act in ACTIONS:
    env.step(act)

env.render() # see the stuck box!
```

**Comment by awjuliani** [comment]
Thanks for bringing this up @unixpickle. We have actually addressed this internally for the Round 2 release. If we can track down and solve the CPU freezing problem you are having before the end of Round 1, we can backport this fix for a new Round 1 release. If not, then it will definitely be fixed in Round 2.

**Comment by awjuliani** [comment]
This should be fixed in the v2.0 release. Please re-open if that is not the case.
